packages/uicore/src/components/Select/Select.tsx

```diff
@@ -21,6 +21,8 @@ export interface SelectOption {
   label: string
   value: string | number | symbol
   icon?: IconProps
+  rightIcon?: IconProps
+  rightIconClassName?: string
```

Comment:
`IconProps` extends `StyledProps` which has `className` so `rightIconClassName` may not be needed